From 6726af19ffeedb200153be57e1b070a415c4bea2 Mon Sep 17 00:00:00 2001
From: Jaegeuk Kim <jaegeuk@google.com>
Date: Wed, 20 May 2026 16:22:05 +0000
Subject: [PATCH 156/223] Revert "Reapply "ANDROID: FIXUP: PCI/pwrctrl: Create
 pwrctrl devices only when CONFIG_PCI_PWRCTRL is enabled""

This reverts commit f6b78e7ce21061002d11cfa6aa090e865ebd3f2b.

The patch breaks Pixel.

Bug: 464938478
Bug: 515084775
Bug: 515099956
Change-Id: I8e38796439633661dbf3227e47b91d80a9dbfe9b
Signed-off-by: Jaegeuk Kim <jaegeuk@google.com>
(cherry picked from commit 715c4a487ab53ba47c6a6c8ff7c992fb156643d2)
---
 drivers/pci/probe.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pci/probe.c b/drivers/pci/probe.c
index b73d8ff9b376..61d2f12996d3 100644
--- a/drivers/pci/probe.c
+++ b/drivers/pci/probe.c
@@ -2455,7 +2455,7 @@ bool pci_bus_read_dev_vendor_id(struct pci_bus *bus, int devfn, u32 *l,
 }
 EXPORT_SYMBOL(pci_bus_read_dev_vendor_id);
 
-#if IS_ENABLED(CONFIG_PCI_PWRCTL)
+#if IS_ENABLED(CONFIG_PCI_PWRCTRL)
 static struct platform_device *pci_pwrctrl_create_device(struct pci_bus *bus, int devfn)
 {
 	struct pci_host_bridge *host = pci_find_host_bridge(bus);
-- 
2.50.1

